app/views/components/blockEditor.html

```diff
@@ -114,34 +113,39 @@
               </label>
             </div>
           </div>
-          <p ng-if="!canChangeRegTypes" class="text-muted"><i class="fa fa-exclamation-triangle"></i> <translate>This question must be shown for all registrant types.</translate></p>
+          <p ng-if="!canChangeRegTypes" class="text-muted"><i class="fa fa-exclamation-triangle"></i>
+            <translate>This question must be shown for all registrant types.</translate>
+          </p>
         </div>
         <div class="form-group" ng-if="hasOptions">
           <label class="inline-label" translate>Expense Type</label>
-             <select class="form-control" ng-model="block.expenseType" ng-options="key as value for (key , value) in expenseTypesConstants">
+          <select class="form-control" ng-model="block.expenseType" ng-options="key as value for (key , value) in expenseTypesConstants">
               <option value="" translate>None</option>
              </select>
         </div>
       </tab>
       <tab heading="Rules" ng-if="canHaveRules">
-        <p class="text-muted"><i class="fa fa-info-circle"></i> <translate>Rules are a great way to hide questions based on the registrant's answers to other questions. All rules created must be met before the question is displayed.</translate>
-          <translate>Rules can depend on questions of</translate> <a href="" tooltip="{{'Multiple Choice, Dropdown, Number, Date, Gender or Year in School' | translate}}" translate>these types</a> <translate>that are above this question.</translate></p>
+        <p class="text-muted"><i class="fa fa-info-circle"></i>
+          <translate>Rules are a great way to hide questions based on the registrant's answers to other questions. All rules created
+            must be met before the question is displayed.</translate>
+          <translate>Rules can depend on questions of</translate> <a href="" tooltip="{{'Multiple Choice, Dropdown, Number, Date, Gender or Year in School' | translate}}"
+            translate>these types</a>
+          <translate>that are above this question.</translate>
+        </p>
 
```

Comment:
Can you leave the style of code you aren't editing alone? I've noticed you've changed quite a few lines throughout these PRs. It makes it harder to review and your style doesn't seem consistent... Sometimes you put everything on new lines. Sometimes you take what was on new lines and put it back on one. Here you've done some combination of both. I guess it probably exceeded your editors linewidth... I don't really want to have to ignore whitespace and look over new line changes when I'm trying to see what you've changed.

I know this project isn't the most consistent to begin with but I don't think you need to go reformatting it.